_posts/2025-02-20-revolutionizing-opensearch-clients-and-documentation-with-a-unified-api-specification.md

```diff
@@ -10,67 +10,71 @@ authors:
+In the near future, the following components will also be generated from the spec:  
 
-The OpenSearch documentation team has started using the OpenSearch Specification to generate many components of the documentation, [starting with the path and query parameters](https://github.com/opensearch-project/documentation-website/pull/8692) of each API function. The handwritten parameter tables in the documentation repository are being replaced by those generated from the spec. Our experts compare the generated tables with the handwritten ones to ensure accuracy and completeness. This process has also helped us correct many overlooked details in the spec.
+- Request and response body fields and their descriptions.  
+- Examples of requests and responses, derived from tests in the spec repository.  
+- Usage examples for each client language.  
 
-In the near future, the team plans to replace the following components with those generated from the spec:
+These changes ensure the documentation remains accurate and consistent with the server and clients, creating a better experience for developers and reducing barriers to OpenSearch adoption.  
 
-* Request and response body shapes and descriptions
-* Request and response examples (from tests in the spec repository)
-* Usage examples for each language client
+## Thank you
 
----
+Over the past year, it has been rewarding to witness the evolution and improvement of the OpenSearch API specification. The specification now serves as the authoritative reference for OpenSearch clients, documentation, and server, ensuring consistency across all components. We look forward to seeing how the OpenSearch project will continue to grow with the OpenSearch API specification as its foundation.
 
-It has been a joy to watch the OpenSearch Specification evolve and improve over the past year. The specification has become the single source of truth for the OpenSearch clients, documentation, and server, bridging the gaps between them. We are excited to see what the future holds for the OpenSearch project with the OpenSearch Specification as its foundation.
+We couldn't have achieved this without the dedication of our contributors. Their feedback, suggestions, and contributions have been invaluable to the project. The Amazon OpenSearch team would like to give a special thanks to the following contributors for their outstanding work on the OpenSearch API specification:
 
-Last but not least, we could not have reached this point without our contributors. Your feedback, suggestions, and contributions have been invaluable to the project. The Amazon OpenSearch team would like to give a special shout-out to the following contributors for their outstanding work on the OpenSearch Specification:
+* [Niyazbek Torekeldi](https://github.com/Tokesh): For his work on completing the OpenSearch API specification by adding missing APIs and fixing numerous spec errors. Tokesh also helped us reach 100% test coverage for the spec.
```

Comment:
The spec is not complete yet, esp with new features from the servers that have yet to be filled. Torekeldi helped us backfill the existing API with missing endpoints and parameters.